src/components/MovieDetails/MovieDetails.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { FC } from 'react';
+
+import './MovieDetails.css';
+import { MovieType } from './../../store/movies/types';
+import Netflixroulette from '../Netflixroulette/Netflixroulette';
+import Button from '../Button/Button';
+import Typography, { TypographyTypes } from '../Typography/Typography';
+import getMovieDuration from './../../helpers/getMovieDuration';
+
+const MovieDetails: FC<MovieType> = ({
+	imageUrl,
+	movieName,
+	releaseYear,
+	genresList,
+	rating,
+	duration,
+	description,
+	onMovieClick,
+}) => {
+	const isRatingHasOneDigit = rating % 1 == 0;
+	return (
+		<>
+			<div className='movie-details-netflixroulette-line'>
+				<Netflixroulette light='light' />
+				<Button
+					buttonClass='go-to-all-movies-button'
+					children='⚲'
+					dataTestid='go-to-all-movies-button'
+					onClick={() => onMovieClick(null)}
+				/>
```

Comment:
`children` is a component inside this component:
```suggestion
				<Button
					buttonClass='go-to-all-movies-button'
					dataTestid='go-to-all-movies-button'
					onClick={() => onMovieClick(null)}
				>⚲</Button>
```